src/Classes/DataPackage.cs

```diff
@@ -337,6 +339,15 @@ private void ProcessAnalyticsLine(string line)
                 case "accepted_instant_invite":
                     if (eventData.GuildId != null) this.AcceptedInvites.Add(eventData);
                     break;
+                case "voice_disconnect":
+                    this.VoiceDisconnections.Add(new DVoiceConnection()
+                    {
+                        ChannelId = eventData.ChannelId,
+                        GuildId = eventData.GuildId,
+                        Timestamp = eventData.Timestamp,
+                        Duration = new TimeSpan(eventData.Duration * 10000)
```

Comment:
This would be more clear
```suggestion
                        Duration = TimeSpan.FromMilliseconds(eventData.Duration)
```